crates/primitives-traits/src/constants/mod.rs

```diff
@@ -10,6 +10,9 @@ pub const RETH_CLIENT_VERSION: &str = concat!("reth/v", env!("CARGO_PKG_VERSION"
 /// Minimum gas limit allowed for transactions.
 pub const MINIMUM_GAS_LIMIT: u64 = 5000;
 
+/// Maximum gas limit allowed for block.
```

Comment:
I think this comment is not completely correct right now-- the limit seems to apply to a transaction (looking for a reference), on the assumption that a transaction can use up all of the gas in the block.

After 7825, this will no longer be the case though

Reply:
Adding this note here because the comment for MINIMUM_GAS_LIMIT refers to a transaction, even though its comparing against `header.gas_limit()`